common/src/main/java/com/wynntils/mc/utils/ComponentUtils.java

```diff
@@ -294,9 +294,9 @@ private static void handleStyleDifference(Style oldStyle, Style newStyle, String
                     result.append(different);
                     return;
                 }
-            }
 
-            result.append(ChatFormatting.RESET);
+                result.append(ChatFormatting.RESET);
```

Comment:
I thought I had commented on this, but I can't see it now. What is this? I don't understand the effect of moving this RESET append. 

The change does not seem to have anything to do with 1.19. If it is an actual bug fix, can you push it separately into main?

Reply:
No, it is a 1.19 bug fix. ComponentUtils#getCoded decoding behaved differently in 1.19, making our regexes fail. I've looked at them, only to find an excessive amount of RESETs. Turns out, something changed internally, and with this change, we get the same results as 1.18.2.

Reply:
I see. Thanks.